assets/pages/bundles-landing/support-landing-ab-test/components/graveErrorNewDesign.jsx

```diff
@@ -0,0 +1,40 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React from 'react';
+
+import { SvgCross } from 'components/svg/svg';
+
+import type { Node } from 'react';
+
+
+// ----- Props ----- //
+
+type PropTypes = {
+  ctaLink: Node,
+  onClose: () => void,
+};
+
+
+// ----- Component ----- //
+
+export default function GraveError(props: PropTypes) {
```

Comment:
Why the name `GraveError`? Does it have a grave shape?

Reply:
That's @Amohkhan's word for it.